xapian-applications/omega/handler_libabw.cc

```diff
@@ -0,0 +1,168 @@
+/** @file handler_libabw.cc
+ * @brief Extract text and metadata using libarchive.
+ */
+/* Copyright (C) 2020 Parth Kapadia
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+#include <config.h>
+#include "handler.h"
+#include "stringutils.h"
+
+#include <memory>
```

Comment:
Is anything from `<memory>` actually used in this file?

Reply:
Thanks for pointing this out. I noticed that some header files were used in the previous versions of the handler but after the changes, these were not required. I have removed them and I will try and make sure to keep a habit of checking if the header files are still required after making some changes.